ocamldoc/odoc_class.ml

```diff
@@ -117,7 +117,7 @@ let rec class_elements ?(trans=true) cl =
     | Class_constraint (c_kind, _ct_kind) ->
         iter_kind c_kind
       (* FIXME : use c_kind or ct_kind ?
-         For now, as ct_kind is not analyzed,
+         For now, as ct_kind is not analysed,
```

Comment:
Same remark as earlier, you should choose between "analyse" and "analyze" once and for all.